pkg/cmd/grafana-cli/commands/conflict_user_command.go

```diff
@@ -256,8 +258,15 @@ func getValidConflictUsers(r *ConflictResolver, b []byte) error {
 		if err != nil {
 			return fmt.Errorf("could not parse the content of the file with error %e", err)
 		}
-		if !previouslySeenEmails[strings.ToLower(newUser.Email)] {
-			return fmt.Errorf("not valid email: %s, email not in previous conflicts seen", newUser.Email)
+		if newUser.ConflictEmail != "" {
```

Comment:
Is there a specific reason to not join these 2 conditionals with `AND`?

Reply:
no real reason. Could refactor into a function call to make it clear what we are after

Reply:
it's a situation of AND/OR for conflicts here.

I will merge this and see if we want to refactor that later.

Made into one if statement